launch/launch/actions/declare_launch_argument.py

```diff
@@ -79,19 +81,47 @@ def __init__(
         name: Text,
         *,
         default_value: Optional[SomeSubstitutionsType] = None,
-        description: Text = 'no description given',
+        description: Optional[Text] = None,
+        choices: List[Text] = None,
         **kwargs
     ) -> None:
         """Create a DeclareLaunchArgument action."""
         super().__init__(**kwargs)
         self.__name = name
+        self.__logger = launch.logging.get_logger(__name__)
         if default_value is None:
             self.__default_value = default_value
         else:
             self.__default_value = normalize_to_list_of_substitutions(default_value)
-        self.__description = description
+        if choices is not None:
+            if choices == []:
```

Comment:
@v-lopez 
```suggestion
            if not choices:
```
? Or 
```suggestion
            if len(choices) == 0:
```
 would do as well.

Reply:
Going with `len`, I want it to throw if for any reason we change the code and choices is `None` and reaches this statement.